pkg/destroy/azure/azure.go

```diff
@@ -226,9 +247,95 @@ func (o *ClusterUninstaller) Run() (*types.ClusterQuota, error) {
 		o.Logger.Debug(err)
 	}
 
+	deadline, _ = waitCtx.Deadline()
+	diff = time.Until(deadline)
+	if diff > 0 {
+		waitCtx, _ = context.WithTimeout(context.Background(), diff) //nolint: govet ctx is thru its parent
```

Comment:
```suggestion
		waitCtx, _ = context.WithTimeout(context.Background(), diff) //nolint:govet ctx is thru its parent
```
My bad, there shouldn't be a space after `nolint:`. But lets wait for the current CI jobs run. I wanna see if the destroy is still working correctly for the regular and shared VPC jobs.

Reply:
fixed. this worked for me locally tho 🤔

Reply:
Looks like the linter cannot be convinced to let this pass. Any other ideas?

Reply:
https://github.com/openshift/installer/pull/7750 has all the tags to merge, just waiting on the tests now.

Reply:
rebased!